src/libAtomVM/opcodesswitch.h

```diff
@@ -1492,9 +1525,38 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
             case OP_ALLOCATE_HEAP_ZERO: {
                 int next_off = 1;
                 int stack_need;
-                DECODE_INTEGER(stack_need, code, i, next_off, next_off);
+                if (UNLIKELY(IS_EXTENDED_ALLOCATOR(code, i, next_off))) {
+                    stack_need = 0;
```

Comment:
Likewise, I could't find a test that runs this block.

Reply:
removed